Making FAQ more mentor specific (#19)
communitybridge/docs: mentorship/mentor-guide/mentorship-faqs.md

# Mentorship FAQs

> **How much time are mentors expect to commit to mentoring?**

Author: ```suggestion
**How much time is expected from Mentors to commit to mentoring?**
```

The Linux Foundation offers mentorship programs around the year with specific term dates. Please check the projects of your interest to find details of the program term dates.

> **How long is a mentors engaged in the program?**

Author: ```suggestion
**How long are mentors engaged in the program?**
```
